R/theme.R

```diff
@@ -194,7 +194,7 @@
 #'   switched (`unit`)
 #'
 #' @param ... additional element specifications not part of base ggplot2. In general,
-#'   these should also be defined in the `element tree` argument.
+#'   these should also be defined in the `element tree` argument. A list is also supported.
```

Comment:
```suggestion
#'   these should also be defined in the `element tree` argument. [Splicing][rlang::splice] a list is also supported.
```
A bare list is not supported, but splicing a list is.